cg/meta/workflow/microsalt.py

```diff
@@ -307,7 +307,7 @@ def resolve_case_sample_id(
     def get_case_id_from_ticket(self, unique_id: str) -> Tuple[str, None]:
         """If ticked is provided as argument, finds the corresponding case_id and returns it.
         Since sample_id is not specified, nothing is returned as sample_id"""
-        case_obj: Family = self.status_db.find_family_by_name(unique_id)
+        case_obj: Family = self.status_db.get_case_by_name(unique_id)
```

Comment:
```suggestion
        case: Family = self.status_db.get_case_by_name(unique_id)
```
Use named args

Reply:
I think we should create separate PR:s where we rename all instances of model_obj, model_object, model_objects, model_objs to just model. But lets do it at a later point when the codebase is more stable.